pkg/sql/control_jobs.go

```diff
@@ -14,11 +14,9 @@ import (
 	"context"
 
 	"github.com/cockroachdb/cockroach/pkg/jobs"
+	jobsauth "github.com/cockroachdb/cockroach/pkg/jobs/authorization"
```

Comment:
maybe just call the package jobs/jobsauth so we don't need to always be aliasing? /2¢